src/main/java/net/bramp/ffmpeg/builder/FFmpegHlsOutputBuilder.java

```diff
@@ -0,0 +1,123 @@
+package net.bramp.ffmpeg.builder;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableList;
+import javax.annotation.CheckReturnValue;
+import static com.google.common.base.Preconditions.*;
+import static net.bramp.ffmpeg.FFmpegUtils.toTimecode;
+import static net.bramp.ffmpeg.Preconditions.checkNotEmpty;
+import java.util.concurrent.TimeUnit;
+
+public class FFmpegHlsOutputBuilder extends FFmpegOutputBuilder {
+
+    public Long hls_time;
+    public String hls_segment_filename;
+    public Long hls_init_time;
+    public Integer hls_list_size;
+    public String hls_base_url;
+
+
+    protected FFmpegHlsOutputBuilder(FFmpegBuilder parent, String filename) {
+        super(parent, filename);
+    }
+
+
+    /**
+     * Set the target segment length. Default value is 2.
```

Comment:
Minor nit. Default value is 2 what? 2 seconds?

Reply:
oh, yes seconds 🤣
I added it
Thank you.